UPDATING.md

```diff
@@ -1994,17 +1994,17 @@ Migrated are:
 | airflow.hooks.S3_hook.S3Hook                                 | airflow.providers.amazon.aws.hooks.s3.S3Hook                    |
 | airflow.contrib.hooks.aws_athena_hook.AWSAthenaHook          | airflow.providers.amazon.aws.hooks.athena.AWSAthenaHook         |
 | airflow.contrib.hooks.aws_lambda_hook.AwsLambdaHook          | airflow.providers.amazon.aws.hooks.lambda_function.AwsLambdaHook         |
-| airflow.contrib.hooks.aws_sqs_hook.SQSHook                   | airflow.providers.amazon.aws.hooks.sqs.SQSHook        |
+| airflow.contrib.hooks.aws_sqs_hook.SqsHook                   | airflow.providers.amazon.aws.hooks.sqs.SqsHook        |
```

Comment:
There is no `SqsHook` in contrib.
We also can't suggest here `airflow.contrib.hooks.aws_sqs_hook.SQSHook` -> `airflow.providers.amazon.aws.hooks.sqs.SqsHook` because this table is for upgrading from 1.10 to 2.0 and in 2.0 only backport amazon provider is compatible so the changes we do in this PR are not relevant for it.

Long story short - please undo the changes you made for this file :)

Reply:
I still see changes on this file
@KennyRich did you undo all changes to this file?

Reply:
Yes